Overflow bug fix and color adjusts

FelipeMendesS opened the pull request:
Overflow bug fix and color adjusts #1

FelipeMendesS committed:
Added support for tables larger than the current screen and scrolling inside these tables

FelipeMendesS committed:
Changed color of the instructions to green

bernardorufino commented on pick.py:
```diff
@@ -7,51 +7,56 @@
-                                                lines are always delimited by \\n.""")
+                                    standard input. The delimiter for the
+                                     columns may be chosen, whereas
+                                    lines are always delimited by \\n.""")
```
Align with first line of string, keep original for lines 14-16

bernardorufino commented on pick.py:
```diff
@@ -7,51 +7,56 @@
 
-parser.add_argument('-d', '--delimiter', default=None,
-                    help="Delimiter to split the columns of the table, defaults to any whitespace char.")
+parser.add_argument('-d', '--delimiter', default=None, help="Delimiter to split the columns of the table, defaults to any whitespace char.")
```
Why?

bernardorufino commented on pick.py:
```diff
@@ -7,51 +7,56 @@
     # Draw instructions
     printstr(stdscr, "[q] abort / [arrows] move / [space] (un)select cell / [d] clear selection / [c] select column", curses.color_pair(3))
-    printstr(stdscr, "[enter] print and copy selected cells (protip: use `pbpaste | ...` to pipe forward)", curses.color_pair(3))
+    printstr(stdscr, "[enter] print and copyselected cells(protip: use `pbpaste | ...` to pipe forward)", curses.color_pair(3))
```
Why no space?

bernardorufino commented on table_view.py:
```diff
@@ -23,10 +23,11 @@ def __init__(self, lines, d):
 
-        :param stdscr: Screen to draw table into
+        :param stdscr: Standard screen that just contains some information
+        :param table_pad: pad to draw table into
```
**P**ad to draw table into

FelipeMendesS committed:
fixed style problems

bernardorufino commented:
I'm ok with the commands being aligned in the center, we can punt the height adjustment to another PR, but please preserve margins
Before:
![image](https://cloud.githubusercontent.com/assets/1025913/9700447/8272f6dc-53d9-11e5-9670-4b96ed7b52f3.png)
After: 
![image](https://cloud.githubusercontent.com/assets/1025913/9700453/9a429416-53d9-11e5-9363-f62b06e4c865.png)
Note the top margin and left margin for command list and input preview

bernardorufino commented on pick.py:
```diff
@@ -24,34 +23,41 @@
     # Draw instructions
     printstr(stdscr, "[q] abort / [arrows] move / [space] (un)select cell / [d] clear selection / [c] select column", curses.color_pair(3))
-    printstr(stdscr, "[enter] print and copy selected cells (protip: use `pbpaste | ...` to pipe forward)", curses.color_pair(3))
+    printstr(stdscr, "[enter] print and copy selected cells(protip: use `pbpaste | ...` to pipe forward)", curses.color_pair(3))
```
Revert `cells (` => `cells(` (Space before open paren)

bernardorufino commented on pick.py:
```diff
@@ -7,13 +7,12 @@
                                  description="""Interactively lets the user pick a cell in a table given to the
-                                                standard input. The delimiter for the columns may be chosen, whereas
+                                                standard input. The delimiter for the
+                                                columns may be chosen, whereas
```
It seems your editor is breaking lines at 80 char limit, we can agree to that but not on this PR, please relax it to 120 chars and then we change everything to 80 in the future if it's the case

bernardorufino committed:
Support margin and fix scroll issues

Original margins weren't supported. Scrolling was broken when going right because
of variable columns lengths. Resizing wasn't working. Also moved scroll code away
from TableView.
